Tools/Pipeline/Controls/BuildOutput.eto.cs

```diff
@@ -11,8 +11,9 @@ public partial class BuildOutput : Pad
     {
         Panel panel;
         TextArea textArea;
-        Scrollable scrollable1;
+        Scrollable scrollable;
```

Comment:
`scrollable1` was not a typo, there were 2 scrollables, tho the other one got axes since textArea already had a scrollable component inside it. I don't mind it being changed.

Reply:
Ooops. I literally cannot tell the difference between 1 and 1. That's also a typo I make a lot, so I was unable to interpret it any other way. Glad it's not a problem